### PR TITLE
Remove ResourceQuota parameter

### PR DESCRIPTION
**Pull request by Byronliu123**
I got the warning info: the parameter resourcequota is obsolete after I executed the powershell of Example2

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Example 1 updates the owner of site collection <https://contoso.sharepoint.com/s
 ### -----------------------EXAMPLE 2-----------------------------
 
 ```powershell
-Set-SPOSite -Identity https://contoso.sharepoint.com/sites/site1 -ResourceQuota 0 -StorageQuota 1024
+Set-SPOSite -Identity https://contoso.sharepoint.com/sites/site1 -StorageQuota 1024
 ```
 
 Example 2 updates the settings of site collection <https://contoso.sharepoint.com/sites/site1.> The storage quota is updated to 1024 megabytes (1 GB) and the resource quota is updated to 0 megabytes.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:94]:
```suggestion
Example 2 updates the settings of site collection <https://contoso.sharepoint.com/sites/site1.> The storage quota is updated to 1024 megabytes (1 GB).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Byronliu123, kindly commit the suggested change. Many thanks.

**Review by dariomws — COMMENTED** [review]
Same comment as https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/165.

**Comment by yogkumgit** [comment]
@Byronliu123 Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@Byronliu123 Gentle reminder.

**Comment by yogkumgit** [comment]
@Byronliu123 Please implement the suggested feedback so that merge can be initiated. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please help.

**Pull request merged**